plugin/src/main/java/org/autorefactor/refactoring/rules/CollectionRefactoring.java

```diff
@@ -398,28 +401,46 @@ private boolean isZero(Expression expr) {
     public boolean visit(IfStatement node) {
         final PrefixExpression pe = as(node.getExpression(), PrefixExpression.class);
         if (hasOperator(pe, NOT)) {
-            final MethodInvocation miContains = as(pe.getOperand(), MethodInvocation.class);
-            return maybeReplaceSetContains(miContains, node.getThenStatement(), pe, false);
+            return maybeReplaceSetContains(node, pe.getOperand(), node.getThenStatement(), node.getElseStatement(),
+                    false);
         } else {
-            final MethodInvocation miContains = as(node.getExpression(), MethodInvocation.class);
-            return maybeReplaceSetContains(miContains, node.getElseStatement(), miContains, true);
+            return maybeReplaceSetContains(node, node.getExpression(), node.getElseStatement(), node.getThenStatement(),
+                    true);
         }
     }
 
-    private boolean maybeReplaceSetContains(
-            MethodInvocation miContains, Statement stmt, Expression toReplace, boolean negate) {
-        if (isMethod(miContains, "java.util.Set", "contains", "java.lang.Object")) {
-            Statement firstStmt = getAsList(stmt, 0);
-            MethodInvocation miAdd = asExpression(firstStmt, MethodInvocation.class);
+    private boolean maybeReplaceSetContains(final IfStatement nodeToReplace, final Expression ifExpression,
+            final Statement statement,
+            final Statement oppositeStatement, final boolean negate) {
```

Comment:
(Do not act on this, I am fixing it as I write the comment)

In the code base, I am shortening "Expression" to "Expr" and "Statement" to "Stmt". I think this is readable enough, but it shortens the length of the lines, thus it makes the code more readable IMO.
So here I renamed the arguments to `ifExpr`, `stmt` and `oppositeStmt`.